src/Components/Home.jsx

```diff
@@ -0,0 +1,167 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-mixed-spaces-and-tabs */
+/* eslint-disable no-tabs */
```

Comment:
- [ ] Kindly remove every eslint disable rule and instead fix the issue in the code.